x/avs/keeper/avs.go

```diff
@@ -39,4 +41,23 @@ func (k *Keeper) GetAVSMinimumSelfDelegation(ctx sdk.Context, avsAddr string) (s
 }
 
 // GetEpochEndAVSs returns the AVS list where the current block marks the end of their epoch.
-// func (k *Keeper) GetEpochEndAVSs(ctx sdk.Context) ([]string, error) {
+func (k *Keeper) GetEpochEndAVSs(ctx sdk.Context) ([]string, error) {
+	var avsList []types.AVSInfo
+	k.IteratAVSInfo(ctx, func(_ int64, epochEndAVSInfo types.AVSInfo) (stop bool) {
+		avsList = append(avsList, epochEndAVSInfo)
+		return false
+	})
+
+	// Check if avsList is not empty
+	if len(avsList) == 0 {
+		return nil, fmt.Errorf("avsList is empty")
```

Comment:
Why return an error instead of simply returning an empty list?

Reply:
fixed